### PR TITLE
docs: add info on how to use the notebooks on our computer

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Notebooks
 Home to jupyter notebooks for Mindee OSS projects
 
+To open and interact with them locally, you can install the [JupyterLab Desktop](https://github.com/jupyterlab/jupyterlab-desktop) available for Linux, macOS, and Windows (free and Open Source).
```
Comment (fg-mindee) [inline README.md:4]: Three things to keep in mind:
- about the content itself, Jupyterlab is a bit heavier / more advanced (better UX yes), but the core thing that is needed is Jupyter notebook to be installed: https://jupyter.org/install
- FYI: this repo is made to store the notebooks, while I configure an automatic generation of those from markdown or RST files (at least for docTR) to reduce their size while validating their format & execution. Meaning that users will interact with those on the repo referring to them: in docTR, we add the reference of the notebook here https://github.com/mindee/doctr/tree/main/notebooks, which is automatically deployed in the documentation over there https://mindee.github.io/doctr/latest/notebooks.html.
- in those references, I took care of adding "Open with Google Colab" link, which will literally open the notebook in Google Colab (free to use, only require a google account to run in the cloud, not on the machine of the user)

So I would suggest to:
- update the installation/usage instructions starting with the less constraining option: Google Colab, and mention that to run it locally, people can use Jupyter notebook or JupyterLab.
- ensuring that the "delivery" part of those notebooks is made by the corresponding repo (for docTR, over here https://github.com/mindee/doctr/tree/main/notebooks)

What do you think?

**Review by fg-mindee — COMMENTED** [review]
Thanks! Added some comments :ok_hand:

File: README.md (hunk 1 above)
Comment (fharper) [reply README.md:4]: Just to be sure, it means that people shouldn't use/read the notebooks here? It's usefull only for people wanted to create or update them, am I right?

if it's the case, I would close this PR, but I would suggest you to update the README to basically have your FYI in it. What do you think?

Comment (fg-mindee) [reply README.md:4]: Yes, see this repo more as a notebook hosting place. They are accessed through the corresponding repo, and generally directly opened in Google Colab :+1: 

This repo is only for creation & updating them, correct!

Excellent idea about the FYI :ok_hand:

Comment (fg-mindee) [reply README.md:4]: Here is the PR #7

**Pull request closed without merging**